cmds/core/ping/ping.go

```diff
@@ -82,18 +87,33 @@ func cksum(bs []byte) uint16 {
 	return ^uint16(sum)
 }
 
-func ping1(netname string, host string, i uint64) (string, error) {
+func ping1(net6 bool, host string, i uint64, waitFor time.Duration) (string, error) {
```

Comment:
Why not just take netname as a parameter?

Reply:
I use `net6` several times below to tell if it's a ping4 or ping6 request, and the netname is just the string to pass to `Dial` (doing the same with netname wouldn't work)

Reply:
(I'll merge this, if you think I should do this differently I'll do that in another PR)